AUTHORS

```diff
@@ -174,6 +180,7 @@ Gavin McNair                            gavin.mcnair@kaluza.com
 George Gaál                             gb12335@gmail.com
 George Kontridze                        gkontridze@plaid.com
 Geyslan G. Bem                          geyslan@gmail.com
+GH action                               ghabot@does.not.exist.cilium.org
```

Comment:
Do we want this?

Reply:
I don't know if we have a way to ignore authors so I'll just leave it for now. cc @joestringer